meb/utils/utils.py

```diff
@@ -101,8 +101,10 @@ class Printer:
     def __init__(self, config, split_column: str):
         self.cf = config
         self.split_column = split_column
-        if self.cf.action_units:
+        if 'AU' in self.cf.action_units[0]:
```

Comment:
This seems dangerous. Need to first check if `self.cf.action_units` is a list and only after that we can index. It could be set to `None`

Reply:
amended in upcoming commit

Reply:
with isinstance check